runtime/onert/core/include/compiler/BackendManager.h

```diff
@@ -61,7 +61,7 @@ class BackendManager
 
 private:
   std::map<std::string, std::unique_ptr<void, dlhandle_destroy_t>> _handle_map;
-  std::map<std::string, std::unique_ptr<backend::Backend, backend_destroy_t>> _gen_map;
+  std::map<std::string, std::unique_ptr<backend::Backend, backend_destroy_t>, std::less<>> _gen_map;
```

Comment:
Is it intended change?

Reply:
Yes, the use of `std::less<>` makes the comparator transparent, allowing the map to perform heterogeneous lookups. This means we can use types like `std::string_view` directly to search the map without having to convert them to `std::string` first.

Reply:
You can see an example here https://en.cppreference.com/w/cpp/utility/functional/less_void